Triage-Process.md

```diff
@@ -0,0 +1,116 @@
+
+In all cases, issues may be closed by maintainers if they don't receive timely response when
+further information is sought, or when additional questions are asked.
+
+## Approaches and Best Practices for getting into triage contributions
+
+Review the organization's [StatusBoard](https://expressjs.github.io/statusboard/),
+pay special attention to these columns: stars, watchers, open issues and contributors.
+This gives you a general idea about the criticality and health of the repository.
+Pick a few projects based on that criteria, your interests, and skills (existing or aspiring).
+
+Review the project's contribution guideline if present. In a nutshell,
+commit to the community's standards and values. Review the
+documentation, for most of the projects it is just the README.md, and
+make sure you understand the key APIs, semantics, configurations, and use cases.
+
+It might be helpful to write your own test apps to re-affirm your
+understanding of the key functions. This may identify some gaps in
+documentation, record those as they might be good PR's to open.
+Skim through the issue backlog; identify low hanging issues and mostly new ones.
+From those, attempt to recreate issues based on the OP description and
+ask questions if required. No question is a bad question!
+
+### Becoming a Triager
+
+Open a pull request in expressjs/express repo to request the triage role.
+Use https://github.com/expressjs/express/pull/4114 for reference. The important
+parts are to state that you have read and agree to the CoC and details of the role,
+that you add your name to the list of triagers (email and pronouns optional), and
+that you follow the commit message format.
+
+Here are examples you can just copy paste:
+
+Issue template:
+
+```
+Title: Request triager role for <your github username>
+
+I have read and understood the project's Contributing guide.
+I also have read and understood the process and best practices around Express triaging.
+
+I request for a triager role for the followting orgs:
+
+jshttp
+pillarjs
+express
+```
+
+Commit Message:
+
+```
+docs: request triager role for <your github username>
+```
+
+Format for name addition to the bottom of this file:
+
+```
+* [<your github username>](https://github.com/<your github username>) - **<Your Display Name>** &lt;<your email>&gt; (<your pronouns>)
+```
+
+Once you have opened your PR, a member of the TC will add you to the `triage` team in the
+orgs requested.  They will then merge your PR and close the issue.  Happy triaging!
+
+### Removal of Triage Role
+
+There are a few cases where members can be removed as triagers:
+
+- Breaking the CoC or project contributor guidelines
+- Abuse or misuse of the role as deemed by the TC
+- Lack of participation for more than 6 months
+
+If any of these happen we will discuss as a part of the triage portion of the regular TC meetings.
+If you have questions feel free to reach out to any of the TC members.
+
+### Other Helpful Hints:
+
```

Comment:
```suggestion
- Everyone is welcome to attend the [Express Technical Committee Meetings](https://github.com/expressjs/discussions#expressjs-tc-meetings), and as a triager it might help to get a better idea of what's happening with the project as well as have your voice heard
```